vigranumpy/test/test_segmentation.py

```diff
@@ -33,6 +33,49 @@ def test_labelMultiArray():
     _impl_test_labelMultiArray(numpy.uint8)
     _impl_test_labelMultiArray(numpy.uint32)
     _impl_test_labelMultiArray(numpy.float32)
+
+
+def _impl_test_applyMapping(dtype):
+    original = numpy.arange(100, dtype=dtype ).reshape(10,10)
+    mapping = { int(k) : int(k + 100) for k in original.flat[:] }
+
```

Comment:
>I've created PR #336 with an implementation. Please check if it solves your problem.

Yes, works perfectly.  Thanks, this will save a lot of little headaches.  BTW, I made a PR onto your PR.  It eliminates the now-obsolete scalar conversions from `test_segmentation.py`.